### ISSUE TITLE
Help On Themes

### ISSUE THREAD

**Issue by Derpitu** [Jiiks/BetterDiscordApp#889]
I do not know how to use the themes part of this code installation, plz help

**Comment by VaporousCreeper** [comment]
What do you mean about _"I do not know how to use the themes part of this code installation"_?
The first thing came to mind was how to install themes.

**Comment by RudySPG** [comment]
yeah i think he mean how do you get themes to work on v2, haven't been able to get them to work either
